analysis/organize_data.qmd

```diff
@@ -5,16 +5,16 @@ title: "Organize Data Notebook"
 ```{r}
 #| label: setup
 #| include: false
-library(tidyverse)
-library(lubridate)
-library(tidytext)
+library(here)
 source(here("utils","check_packages.R"))
 source(here("utils","functions.R"))
+```
 
+```{r}
+#| label: read-data
 
 # reading in #joebiden data
-
-tweets_biden <- read.csv("data/data_raw/tweets_csv/hashtag_joebiden.csv.gz",
+tweets_biden <- read_csv(here("data", "data_raw", "tweets_csv", "hashtag_joebiden.csv.gz"),
                          na = "NA") |>
```

Comment:
Lets use `read_csv` (tidyverse) not `read.csv` (base). Its more robust. Also I switched the pathway approach to use `here` which is also more robust.